sympy/core/relational.py

```diff
@@ -381,12 +381,15 @@ class Equality(Relational):
 
     is_Equality = True
 
-    def __new__(cls, lhs, rhs=0, **options):
+    def __new__(cls, lhs, rhs=None, **options):
         from sympy.core.add import Add
         from sympy.core.logic import fuzzy_bool
         from sympy.core.expr import _n2
         from sympy.simplify.simplify import clear_coefficients
 
+        if rhs is None:
+            raise ValueError("rhs must be provided")
```

Comment:
The correct error here would be `TypeError`, which will happen automatically if you remove the default keyword argument from `rhs` and make it a required argument.